From: Bjorn Helgaas <bhelgaas@google.com>
Date: Mon, 25 Mar 2019 13:36:52 -0500
Subject: i2c: isch: Remove unnecessary acpi.h include
Git-commit: 0ec4b562c378ea2f1da7a5aa1b783114b84fed5f
Patch-mainline: v5.2-rc1
References: git-fixes

Commit 54fb4a05af0a ("i2c: Check for ACPI resource conflicts") included
<linux/acpi.h> so we could use acpi_check_region(). Commit fd46a0064af1
("i2c: convert i2c-isch to platform_device") removed the use of
acpi_check_region() but not the include.

Remove the now-unnecessary include of <linux/acpi.h>.  No functional change
intended.

Fixes: fd46a0064af1 ("i2c: convert i2c-isch to platform_device")
Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
Reviewed-by: Jean Delvare <jdelvare@suse.de>
Reviewed-by: Mukesh Ojha <mojha@codeaurora.org>
Signed-off-by: Wolfram Sang <wsa@the-dreams.de>
Acked-by: Nicolas Saenz Julienne <nsaenzjulienne@suse.de>
---
 drivers/i2c/busses/i2c-isch.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/i2c/busses/i2c-isch.c b/drivers/i2c/busses/i2c-isch.c
index 5c754bf659e2..f64c1d72f73f 100644
--- a/drivers/i2c/busses/i2c-isch.c
+++ b/drivers/i2c/busses/i2c-isch.c
@@ -30,7 +30,6 @@
 #include <linux/ioport.h>
 #include <linux/i2c.h>
 #include <linux/io.h>
-#include <linux/acpi.h>
 
 /* SCH SMBus address offsets */
 #define SMBHSTCNT	(0 + sch_smba)

